pkg/gofr/datasource/pubsub/kafka/kafka_test.go

```diff
@@ -296,7 +302,7 @@ func TestNewKafkaClient(t *testing.T) {
 			config: Config{
 				Broker: "kafka-broker",
 			},
-			expected: false,
+			expected: true,
```

Comment:
since, expected is `true` in both the cases.. we can remove it and consider it true by default

Reply:
yups sure will remove this field